src/amo/components/Header/index.js

```diff
@@ -105,7 +105,7 @@ export class HeaderBase extends React.Component {
               <DropdownMenuItem>
                 <Link
                   className="Header-user-menu-collections-link"
-                  href={`/collections/${siteUser.username}/`}
+                  href="/collections/"
```

Comment:
I think the prop should be `to` when we'll enable the collection feature, so maybe you want to do the same as for the user profile:

https://github.com/mozilla/addons-frontend/blob/1e08b34c1b7a62c57e48ee2229bc6eeec9e39b70/src/amo/components/Header/index.js#L68-L70